torch/_inductor/codegen/wrapper.py

```diff
@@ -649,8 +649,20 @@ def generate_profiler_mark_wrapper_call(self, stack):
         )
         stack.enter_context(self.wrapper_call.indent())
 
-    def generate_kernel_call(self, name, call_args, device_index=None, cpp=False):
-        self.writeline(self.wrap_kernel_call(name, call_args))
+    def generate_kernel_call(
+        self, name, call_args, grid=None, device_index=None, cuda=None
```

Comment:
what type is `cuda`? You are passing `cuda=True`, so default should probably be `False`, not `None`?

Reply:
I will change it to `cuda=True` as the triton backend is the default one.